SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -143,19 +139,19 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	UPROPERTY()
 	USpatialStaticComponentView* StaticComponentView;
 	UPROPERTY()
-	USnapshotManager* SnapshotManager;
-	UPROPERTY()
 	USpatialMetrics* SpatialMetrics;
 	UPROPERTY()
 	ASpatialMetricsDisplay* SpatialMetricsDisplay;
 	UPROPERTY()
 	ASpatialDebugger* SpatialDebugger;
 	UPROPERTY()
-	USpatialLoadBalanceEnforcer* LoadBalanceEnforcer;
-	UPROPERTY()
 	UAbstractLBStrategy* LoadBalanceStrategy;
 
-	TUniquePtr<SpatialVirtualWorkerTranslator> VirtualWorkerTranslator;
+	TSharedPtr<SpatialDispatcher> Dispatcher;
```

Comment:
Putting this in a comment so that it can be responded too more easily. Discussed offline that we should try to make these unique pointers if possible provided the pr doesn't become unreadable

Reply:
Updated.